src/einsteinpy/tests/test_metric/test_schwarzschild.py

```diff
@@ -41,7 +41,7 @@
             [50 * u.km, np.pi / 2 * u.rad, np.pi / 2 * u.rad],
             [0.1 * _c * u.m / u.s, 2e-7 * _c * u.rad / u.s, 1e-5 * u.rad / u.s],
             0 * u.s,
-            5.972e24 * u.g,
+            5.972e24 * u.kg,
```

Comment:
No need to change test data. Kindly revert it to `u.g`

Reply:
wait

Reply:
> 
> 
> No need to change test data. Kindly revert it to `u.g`

sorry  i just thought i did that by mistake